.github/workflows/maven.yaml

```diff
@@ -78,7 +94,7 @@ jobs:
         uses: ./.github/workflows/composites/fabric8-istio-integration-test
 
   test:
-    needs: [ build, fabric8_istio_integration_test ]
```

Comment:
we do not need this `fabric8_istio_integration_test`, because it runs one at a time anyway, we do not run the steps in parallel.